sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/regexpExpressions.scala

```diff
@@ -198,6 +200,35 @@ case class StringSplit(str: Expression, pattern: Expression)
   override def prettyName: String = "split"
 }
 
+/**
+ * Splits a string into arrays of sentences, where each sentence is an array of words.
+ * The 'lang' and 'country' arguments are optional, and if omitted, the default locale is used.
```

Comment:
Yep. It's Hive behavior. Hive supports `sentences(str)`, `sentences(str,lang)`, and `sentences(str,lang,country)`.
And Hive does not raise exception for the illegal `lang` and `country`.